getting-started/scaling-out.md

```diff
@@ -0,0 +1,159 @@
+# Scaling out using Distributed Hypertables
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives.
+Hypertables are used to handle large amount of data by breaking it up
+into smaller pieces (chunks), allowing operations to execute efficiently. When
+the amount of data is expected to be beyond what a single machine can handle,
+you can distribute the data chunks over several machines by using
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+In the multi-node topology ([architecture][]), all nodes are TimescaleDB instances.
+The data are distributed and stored in TimescaleDB instances, called data nodes.
+The distributed data are accessed through a distributed hypertable
+on another TimescaleDB instance, called the *access node*.
+The access node is the entry point for any access to data in the cluster.
+
+## Working with Data Nodes
+
+Data nodes act as the containers for the hypertable chunks and are
+necessary to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that:
+
+* Data nodes are added as objects locally and just connect to an
+  existing database server.
+
+* You should already have a running database server on some host.
+
+When creating the data node, you should:
+
+* Provide a name to use when referring to the data node from
+  this database.
+
+* Provide the host where the hypertable partition for the distributed
+  hypertable should be stored.
+  
+* Provide password, which will be used during access to
+  the created remote data node.
+  
+* Provide bootstrap user and password, which is used to
+  create the data node.
+  
+* Ensure that the bootstrap user used for connecting to the data node has
+  `CREATEDB` privilege or is a superuser. 
+  This is necessary since
+  `add_data_node` expects to be able to create a
+  database on the remote data node and create
+  TimescaleDB extension in it.
+
+```sql
```

Comment:
Add point:

Any additional users that will access a distributed hypertable currently need their own user mappings per data node with a `user` and `password` option. A user mapping can be created for data node as follows: 
```
CREATE USER MAPPING FOR foo SERVER <data node> OPTIONS (user 'foo' password 'bar');
```
Additional users also need `USAGE` permission on the `timescaledb_fdw` foreign data wrapper and any data node (server) objects

Reply:
IMHO it is too detailed for the getting started. I added a tip for this. Is it enough?

Reply:
I agree that it is detailed, but I rather have this documented, here or elsewhere (we should probably have these requirements also in add_data_node docs), since it is a requirement. A tip works for additional users. Currently, however, I see server permissions missing, although, obviously, the user who creates the server will have permissions by default. 

Generally, though, I think it is bad if the user hits a snag and doesn't know how to proceed. The proper way to address this in the long term is to simplify the UX in a way that is consistent with a reasonable level of security. Then hopefully this will go away.

Reply:
Done